### PR TITLE
Separete views

### PR DESCRIPTION
**Pull request by ardiadrianadri**
Step-2

### CODE REVIEW

**Comment by raguilera82** [comment]
Are you sure merge the same README.md in master? I think you are overriding the content. If we want a branch per good-practice, we must not merge in master. Please, clarify your idea to organize the information.

**Comment by bytelovers** [comment]
@raguilera82 the Adrian's idea is create a tag for each step of refactorization of code for the users can detach the commits and watch de differences.

**Comment by ardiadrianadri** [comment]
The README is different for each tag. As you can see there are already two tag: step-0 and step-1. Each tag has a different README which explain the changes commited in this git node. The user should reset its git branch in each step to follow the workshop. It is the possibility, as @bytelovers said, to check the changes in the code with a git diff between two tags.

**Comment by raguilera82** [comment]
Ok, now I understand you. Then I think the best way is not merging with master and release a tag else release a tag from each branch ex- (step1-separate-views) and in master we have a README with a list of good practices and its steps. What do you think about this?

**Comment by ardiadrianadri** [comment]
I think that, what @raguilera82 propose is to make a branch for each step and the tag mark the last stable version of this branch so, if we have pull request in one of the steps it'll be easier to check for new updates/ changes and, at the same time, the people who is doing the workshop will be able to reach the last stable version.

**Comment by raguilera82** [comment]
That is the idea @ardiadrianadri

**Comment by raguilera82** [comment]
The other part of the idea is making an index in README of master branch with the steps and good practices references

File: README.md [hunk 1]
```diff
@@ -20,3 +18,6 @@ It is not a good idea to make a deep folder tree in the project. Usually the peo
 
  ### Ruben Aguilera:
  Regular technical speaker, Angular and Polymer expert, developer trainers and advanced software architect. Ruben had worked on many projects, as a full stack developer. Currently working in Spain for Autentia and wrote all his lessons learned in the book ["Angular: guía práctica"(only in Spanish... sorry)](https://leanpub.com/angular-guia-practica)
+
+### Alfredo Delacalle:
```
Comment (bytelovers) [inline README.md]: I would appreciate if you write my last name as **"de la Calle"** xD

Comment (ardiadrianadri) [reply README.md]: Sorry... :(

**Comment by ardiadrianadri** [comment]
I'll do as soon as I finished. True is that I have an idea of the steps.
step -2: Modules and lazy loading
step- 3: Separate logic of service from view logic
step- 4: Configuration of services
step- 5: Common components
step- 6: Inmutable views

But... I won't be sure until I finish...

**Comment by bytelovers** [comment]
I think that separation of logic services (step-3) would be in 2nd place, because in the step-1 is the separation of view and controller, isn't it?

**Comment by ardiadrianadri** [comment]
The services that are need in the example are used in both views so they should be included in the core modules soooooo.... before separate the services from the view I had to introduce the modules.

**Comment by bytelovers** [comment]
Ok, now I understand you

**Pull request closed without merging**